zerver/openapi/zulip.yaml

```diff
@@ -12125,6 +12125,48 @@ paths:
                         description: |
                           An example JSON error response for when the user doesn't have permission
                           to subscribe other users to streams and `stream_ids` is not empty.
+  /invites/{prereg_id}:
+    delete:
+      operationId: revoke-email-invitation
+      summary: Revoke email invitation
+      tags: ["invites"]
+      description: |
+        Revoke an [invitation](/help/invite-new-users#send-email-invitations)
+        that has been sent to the email address of the invitee.
+
+        The non-admin users can only revoke the invitation that they have sent. The
+        administrators can revoke any invitation except the ones that have
+        `invited_as` parameter as `100` (i.e. user created after the acceptance of
+        the invitation will be assigned the [organization-level role](/help/roles-and-permissions)
+        as owner). Only the owners in the organization can revoke such invitations.
+      parameters:
+        - name: prereg_id
+          in: path
+          description: |
+            The ID of the email invitation to be revoked.
+          schema:
+            type: integer
+          example: 1
+          required: true
+      responses:
+        "200":
+          $ref: "#/components/responses/SimpleSuccess"
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - allOf:
+                      - $ref: "#/components/schemas/CodedError"
+                      - example:
+                          {
+                            "result": "error",
+                            "msg": "No such invitation",
+                            "code": "BAD_REQUEST",
+                          }
+                        description: |
+                          A typical failed JSON response for when the `prereg_id` is invalid.
```

Comment:
Let's use wording more similar to what's used in the delete a user group error: "An example JSON error response for an invalid user group ID:"